npc/other/mercenary_rent.txt

```diff
@@ -224,7 +224,7 @@ pay_arche,99,167,4	duplicate(Mercenary Manager#main)	Mercenary Manager#Bow	4_M_J
 	mes "you're looking for?";
 	next;
 	setarray .@item, 12184,	12185,	12241,	12242,	12243;
-	setarray .@cost, 1750,	3000,	560,	1050,	2100;
+	setarray .@cost, 2500,	5000,	800,	1500,	3000;
```

Comment:
Is this different between renewal and pre-renewal?
Then we would need a checkre call, since this is a script that is shared between both modes.

Reply:
I guess @Playtester can check it in pre-re!

Reply:
@Lemongrass3110 @Haydrich 
These are the official prices for pre-renewal:
Mercenary Red Potion - 1750z
Mercenary Blue Potion - 3500z
Mercenary Concentration Potion - 560z
Mercenary Awakening Potion - 1050z
Mercenary Berserk Potion - 2100z

Seems only Blue Potion was wrong in the old pricing for pre-renewal.

Reply:
As @Playtester said above. The prices of kRO seems to be common used in pre-re servers.
Im going to revert the changes.